### ISSUE TITLE
[Bug]: Crash after some seconds of execution on Fedora 40

### ISSUE THREAD

**Issue by hardBSDk** [NoCrypt/migu#50]
### Preflight checklist

- [X] I have searched the [issue tracker](https://github.com/NoCrypt/migu/issues) for a bug report that matches the one I want to file, without success.
- [X] I have searched the [frequently asked questions](https://github.com/NoCrypt/migu/blob/master/docs/faq.md) for a solution that fixes this problem, without success.
- [X] I have checked that I'm using the [latest stable](https://github.com/NoCrypt/migu/releases/latest) version of the app.

### What app version are you using?

5.6.2

### What operating system version are you using?

Linux

### Expected Behavior

Don't crash.

### Actual Behavior

Crashes using the AppImage, can be related to https://github.com/ThaUnknown/miru/issues/564

### Screenshots

None.

**Comment by AshtakaOOf** [comment]
Launch Migu from the terminal and send the log.
Also share what gpu Migu is running on, as well as other relevant info.

**Comment by hardBSDk** [comment]
@AshtakaOOf The GPU is AMD Radeon with the RadeonSI and RADV drivers.

The log:

```
06:53:29.732 › Checking for update
06:53:31.869 › Update for version 5.6.2 is not available (latest version: 5.6.2, downgrade is disallowed).
06:54:06.877 › Error with Permissions-Policy header: Unrecognized feature: 'ch-ua-form-factors'.
06:54:06.884 › Slow network is detected. See https://www.chromestatus.com/feature/5636954674692096 for more details. Fallback font will be used while loading: https://fonts.gstatic.com/s/roboto/v18/KFOmCnqEu92Fr1Mu4mxK.woff2
06:54:06.885 › Slow network is detected. See https://www.chromestatus.com/feature/5636954674692096 for more details. Fallback font will be used while loading: https://fonts.gstatic.com/s/roboto/v18/KFOlCnqEu92Fr1MmEU9fBBc4.woff2
06:54:08.033 › Third-party cookie will be blocked. Learn more in the Issues tab.
06:54:08.033 › Third-party cookie will be blocked. Learn more in the Issues tab.
06:54:08.036 › Third-party cookie will be blocked. Learn more in the Issues tab.
06:54:08.036 › Third-party cookie will be blocked. Learn more in the Issues tab.
06:54:08.316 › Third-party cookie will be blocked. Learn more in the Issues tab.
[444260:1023/065445.576594:ERROR:gpu_process_host.cc(989)] GPU process exited unexpectedly: exit_code=133
[444260:1023/065447.347292:ERROR:gpu_process_host.cc(989)] GPU process exited unexpectedly: exit_code=133
[444260:1023/065455.204863:ERROR:gpu_process_host.cc(989)] GPU process exited unexpectedly: exit_code=133
[444260:1023/065456.146387:ERROR:gpu_process_host.cc(989)] GPU process exited unexpectedly: exit_code=133
[444260:1023/065457.075500:ERROR:gpu_process_host.cc(989)] GPU process exited unexpectedly: exit_code=133
[444260:1023/065458.007261:ERROR:gpu_process_host.cc(989)] GPU process exited unexpectedly: exit_code=133
[444260:1023/065458.617422:ERROR:gpu_process_host.cc(989)] GPU process exited unexpectedly: exit_code=133
[444260:1023/065459.222389:ERROR:gpu_process_host.cc(989)] GPU process exited unexpectedly: exit_code=133
[444260:1023/065459.821851:ERROR:gpu_process_host.cc(989)] GPU process exited unexpectedly: exit_code=133
[444260:1023/065459.821876:FATAL:gpu_data_manager_impl_private.cc(448)] GPU process isn't usable. Goodbye.
Trace/breakpoint trap (core dumped)
```

**Comment by AshtakaOOf** [comment]
From what I can find about this error online. 
Try deleting the following files at `$HOME/.config/Migu` 
```
GPUCache
VideoDecodeStats
Cache
Code Cache
ephemeral.json
DawnCache
```
You couldd also attempt to open it with `--disable-accelerated-video-decode`
